pkg/api/validation/validation.go

```diff
@@ -30,12 +30,32 @@ import (
 	"github.com/golang/glog"
 )
 
+const minPort int = 1
+const minHostPort int = 0
+const maxPort int = 65536
+
+const QualifiedNameErrorMsg string = "must match regex [" + util.Dns1123SubdomainFmt + " / ] " + util.Dns1123LabelFmt
+const PortNameErrorMsg string = "must be less than %d characters and match regex " + util.Dns1123LabelFmt
+const CIdentifierErrorMsg string = "must match regex " + util.CIdentifierFmt
+const NegativeErrorMsg string = "value cannot be negative"
+
+func intervalErrorMsg(lo, hi int) string {
+	return fmt.Sprintf("must be greater than %d and less than %d", lo, hi)
+}
+
+var DnsSubdomainErrorMsg string = "must match regex " + util.Dns1123SubdomainFmt + " and have at most " + fmt.Sprintf("%d", util.Dns1123SubdomainMaxLength) + "characters"
```

Comment:
I think I like the other wording better "must be less than %d characters and match regex %s".  And since we're falling on Sprintf anyway, just do one big Sprintf call rather than some string catenations and some sprintf.